internal/subscriber/ordering_test.go

```diff
@@ -140,6 +222,20 @@ func getDummyOrderedReceivedMessage() []messagebroker.ReceivedMessage {
 	return []messagebroker.ReceivedMessage{msgProto}
 }
 
+func getReceivedMessage() messagebroker.ReceivedMessage {
```

Comment:
why are we not using getDummyOrderedReceivedMessage func..?

Reply:
removed using getDummyOrderedReceivedMessage only now